<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Endoscope::FontBrowser</class>
 <widget class="QWidget" name="Endoscope::FontBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
       <layout class="QVBoxLayout" name="verticalLayout_15">
        <item>
         <widget class="QSplitter" name="splitter_10">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QTreeWidget" name="fontTree">
           <property name="columnCount">
            <number>2</number>
           </property>
           <column>
            <property name="text">
             <string>Fonts</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Smooth sizes</string>
            </property>
           </column>
          </widget>
          <widget class="QWidget" name="layoutWidget_7">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QTableView" name="selectedFontsView"/>
            </item>
            <item>
             <widget class="QLineEdit" name="fontText"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
